scripts/ci-frontend-metrics.sh

```diff
@@ -6,6 +6,7 @@ ACCESSIBILITY_ERRORS="$(grep -oP '\"errors\":(\d+),' pa11y-ci-results.json | gre
 DIRECTIVES="$(grep -r -o  directive public/app/ | wc -l)"
 CONTROLLERS="$(grep -r -oP 'class .*Ctrl' public/app/ | wc -l)"
 LEGACY_FORMS="$(grep -r -oP 'LegacyForms;' public/app | wc -l)"
+CLASSNAME_PROP="$(grep -r -o -E --include="*.ts*" "\.*.className\W:.*;" public/app | wc -l)"
```

Comment:
do you want to include test files here?

Reply:
Currently, we just want to look for `className` in props in `public/app`.

Reply:
@eledobleefe and test files? is that something that's relevant for your stats too?

Reply:
If there are any classNames in test files, I think that information would be highly relevant, as it is not expected.

Reply:
@tskarhed I agree knowing if there are any `classNames` in test files could be interesting but we are just counting the number of files using `className` as a prop, so we don't have a way of distinguishing if that file is a test or not.